pkg/webhook/instance_admission_test.go

```diff
@@ -299,15 +299,15 @@ func TestValidateUpdate(t *testing.T) {
 			wantErr: true,
 		},
 		{
-			name: "parameter update triggering a non-existing OV plan IS allowed but will NOT trigger a plan",
+			name: "parameter update triggering a non-existing OV plan IS NOT allowed",
```

Comment:
I think this change of behaviour deserves at least a mention in the PR description, if not title.

Reply:
It is mentioned in the PR description and somewhat vaguely in the PR title. Do you have a better title suggestion that doesn't exceed 60 characters?

Reply:
This test claims to be about "non-exisent **plans**" while the PR title/description only seems to mention non-existent **parameters**. To me these are two different things.
See also https://github.com/kudobuilder/kudo/issues/1268 w.r.t the former.

Reply:
You're right, plan description is misleading, it's the parameter existence that is being checked. Fixed the description.

Reply:
...and also added a test for the missing **plan** 👍